packages/react/src/components/Authenticator/ConfirmSignIn/ConfirmSignIn.tsx

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import { translate } from '@aws-amplify/ui';
+import { authenticatorTextUtil } from '@aws-amplify/ui';
```

Comment:
All React Authenticator changes are for using `authenticatorTextUtil` instead of hardcoded string.